lib/libutils/isoc/include/malloc_flags.h

```diff
@@ -31,5 +31,10 @@
  * from hi address to low address.
  */
 #define MAF_HI_ALLOC	0x10
+/*
+ * Used by phys_mem_alloc_flags() to indicate whether physical memory
+ * should be allocated from the Core or TA physical memory pool.
+ */
+#define MAF_CORE_MEM	0x11
```

Comment:
0x20 (not zero-init + hi-alloc')

Reply:
Yes, I'll fix this flag and the others below.